app/services/action_builder.rb

```diff
@@ -44,41 +44,22 @@ def member
     return @user if @user.present?
     @user = existing_member || Member.new(email: @params[:email])
 
-    update_member_fields
```

Comment:
It'd be great to get ActionBuilder refactoring into a PR of its own tbh because this is so big. I'm having a bit of a hard time following around.